webpack/assets/javascripts/react_app/components/AutoCompleteSearch/auto-complete.scss

```diff
@@ -0,0 +1,33 @@
+@import '~patternfly/dist/sass/patternfly/_color-variables.scss';
+@import '~patternfly-react/dist/sass/_type-ahead-select.scss';
```

Comment:
Does the `_type-ahead-select.scss` actually needed?

Reply:
Yes, it must have styles.. otherwise things will break.

Reply:
So why doesn't `patternfly-react` import it by default?